openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.interface.ts

```diff
@@ -28,6 +28,7 @@ export interface ChartsPermissions {
 export interface DashboardDetailsProps {
   charts: Array<ChartType>;
   dashboardDetails: Dashboard;
+  fetchDashboard: () => void;
```

Comment:
Why do we need this?

Reply:
yes, we do need this to generate new data after manipulating the task entity.